torch/_inductor/codegen/common.py

```diff
@@ -547,8 +566,9 @@ def store(name, index, value, mode=None):
                 self.store_buffer_names.add(name)
                 if mode is None:
                     self.cse.store_cache[name] = value
-                    for other_name in self.current_node.get_mutations():
-                        self.cse.store_cache[other_name] = value
+                    if self.current_node:
```

Comment:
same here, and below - how did these calls work before that we now have to change to None checking?

Reply:
We run algorithm selection before scheduling, so need to codegen benchmarking code before these are populated.